core/src/main/scala/kafka/coordinator/group/OffsetConfig.scala

```diff
@@ -24,7 +24,7 @@ import kafka.message.{CompressionCodec, NoCompressionCodec}
  * @param maxMetadataSize The maximum allowed metadata for any offset commit.
  * @param loadBufferSize Batch size for reading from the offsets segments when loading offsets into the cache.
  * @param offsetsRetentionMs After a consumer group loses all its consumers (i.e. becomes empty) its offsets will be kept for this retention period before getting discarded.
- *                           For standalone consumers (using manual assignment) offset of a partition will be expired when this retention period is passed since its last commit.
+ *                           For standalone consumers (using manual assignment) offsets of a partition will be expired when this retention period is passed since its last commit.
```

Comment:
Sorry, this still reads a little funny to me. How about this?

> For standalone consumers (using manual assignment), offsets will be expired as soon as the retention period following the time of the commit passes.

Reply:
I'm not sure how accurate we need to be here. The fact is it's not really "as soon as" because `offsets.retention.check.interval.ms` plays a factor too. How about this?

> For standalone consumers (using manual assignment), offsets will be expired after the time
> of last commit plus this retention period.

Reply:
Sounds good.